### DISCUSSION TITLE
exif orientation not changing on rotate

### DISCUSSION THREAD

**Discussion by pgscannell** [ImageMagick/ImageMagick#2353] [Help]
When I resize an image and include the -rotate -90 command, the exif orientation value is still '6', even though the image got rotated.

What do I have to do to fix that?

Thanks,
Paul

**Comment by snibgo** [comment]
I suggest using "-auto-rotate".

EDIT to add: if you want to explicitly set the orientation flag, exiftool can do that, for example:
```
exiftool -n -Orientation=1 myfile.jpg
```

**Comment by pgscannell** [comment]
Where do you find the program "exiftool"?

**Reply by t246246** [reply]
https://exiftool.org/

**Comment by pgscannell** [comment]
I tried the -auto-rotate like you suggewsted, but the file's exif orientation value is still '6'.  As soon as you can tell me where to find "exiftool" I will be abnle to do that next.  I have a feeling that there is another download available that I'm not aware of.  My version ov magick is 1.0.10-23 and I just downloaded it on 7/18/2020 and the name of the file I downloaded and ran is : ImageMagick-7.0.10-23-Q16-HDRI-x86-static.exe

**Comment by pgscannell** [comment]
Corfrection.  My magick version is 7.0.10-23.

**Comment by snibgo** [comment]
> I tried the -auto-rotate like you suggewsted, but the file's exif orientation value is still '6'. 

"-auto-rotate" should read the orientation tag; if it isn't 1, it should rotate the image and set the flag to 1.

If it doesn't work for you, please link to a sample input file, and show the exact command you used.

exiftool can be downloaded from https://exiftool.org/
